generatebundlefile/hack/docker-ecr-config.json

```diff
@@ -7,6 +7,7 @@
         "382577505035.dkr.ecr.us-west-2.amazonaws.com": "ecr-login",
         "646717423341.dkr.ecr.us-west-2.amazonaws.com": "ecr-login",
         "783794618700.dkr.ecr.us-west-2.amazonaws.com": "ecr-login",
+        "067575901363.dkr.ecr.us-west-2.amazonaws.com": "ecr-login",
```

Comment:
do we need all the other registries added here too?

Reply:
Since we're building only to this account, this should suffice. The replication happens outside of this Go code.